src/newdb.cpp

```diff
@@ -1218,7 +1218,7 @@ static bool save_char(char_data *player, DBIndex::vnum_t loadroom, bool fromCopy
   /* Re-equip equipment. */
   for (i = 0; i < NUM_WEARS; i++) {
     if (char_eq[i])
-      equip_char(player, char_eq[i], i);
+      equip_char(player, char_eq[i], i, FALSE);
```

Comment:
Same deal here, you must have a guaranteed `affect_total()` run after this to ensure the character is set back up properly.

Reply:
There's an affect_total a few lines further down.

Reply:
Ah, just below the code fold on GitHub. Thanks!